test/integration_test/kubevirt_hyperconv_test.go

```diff
@@ -1043,3 +1044,23 @@ func waitForVolumeAttachmentsToMove(t *testing.T, testState *kubevirtTestState,
 	// verify that vm stayed up
 	verifyVMStayedUp(t, testState)
 }
+
+func restartVolumeDriverAndWaitForReady(t *testing.T, attachedNode *node.Node) {
+	log.Infof("Restarting volume driver on node %s", attachedNode.Name)
+	err := volumeDriver.RestartDriver(*attachedNode, nil)
+	require.NoError(t, err)
+
+	// RestartDriver function above just labels the k8s node. We need to wait till PX goes down.
+	log.Infof("Sleeping to let the volume driver go down on node %s", attachedNode.Name)
+	time.Sleep(1 * time.Minute)
+
+	require.Eventuallyf(t, func() bool {
+		ret := volumeDriver.IsPxReadyOnNode(*attachedNode)
```

Comment:
Can we use `volumeDriver.WaitForPxPodsToBeUp(*attachedNode)` instead of require.Eventually ?